### DISCUSSION TITLE
Run Mac OS with AMD processor

### DISCUSSION THREAD

**Discussion by wiqoss** [ggml-org/llama.cpp#6543] [Q&A]
How can i run mac os with Ryzen 5 5600X and GTX 970? I can buy another GPU for Mac OS, if it needed

**Comment by beebopkim** [comment]
Why macOS? For your hardware, linux is better.

**Comment by TomDev234** [comment]
It is possible to run macOS on a Ryzen cpu with some limitations (no virtual machines, problems with Adobe software).
Nvidia graphics cards are not supported. 
You need an AMD Radeon™ 6600, 6600 XT, 6800, 6800 XT or a 6900 XT graphics card.
Google for Ryzen Hackintosh or Ryzentosh.
